src/frameworks/remix.json

```diff
@@ -3,12 +3,14 @@
   "name": "Remix",
   "category": "static_site_generator",
   "detect": {
-    "npmDependencies": ["remix", "@remix-run/netlify"],
+    "npmDependencies": ["remix", "@remix-run/netlify", "@remix-run/netlify-edge"],
     "excludedNpmDependencies": [],
     "configFiles": ["remix.config.js"]
   },
   "dev": {
-    "command": "remix watch"
+    "command": "remix watch",
+    "port": 3000,
```

Comment:
[boulder] this must NOT be included. Remix doesn't have its own dev server, so including this will prevent it from running. We added support for frameworks that don't have dev servers specifically for Remix. What we do is run `remix watch` which watches the files, but it uses the Netlify static file server instead.